add_users.py

```diff
@@ -0,0 +1,44 @@
+from app.db import get_session
+from app.models.users import Users
+from sqlmodel import select
+from app.auth.security import get_password_hash
+
+def add_users(users_list_dict: dict):
+    """
+    This function creates new users in the spun up local database
+
+    Args:
+        users_list_dict: Should contain a list of dictionary 
+        usernames and plain text passwords for users
+
+    Raises:
+        Print: If username already exists will print out warning
+        and ignore adding this user.
+    """
+    with next(get_session()) as session:
+        for user_info in users_list_dict:
+            username = user_info.get('username')
+            password = user_info.get('password')
+
+            if not username or not password:
+                print(f"Skipping user with missing username or password: {user_info}")
+                continue
+
+            # Check if the username already exists
+            existing_user = session.exec(select(Users).where(Users.username == username)).first()
```

Comment:
As username is the primary key should this not be replaced with `session.get(Users, username)`